python/3d/auto_generated/qgs3dmapsettings.sip.in

```diff
@@ -79,6 +83,29 @@ to improve the precision.
     QgsVector3D origin() const;
 %Docstring
 Returns coordinates in map CRS at which 3D scene has origin (0,0,0)
+%End
+
+    void setOrientation( double newOrientationZ );
+%Docstring
+Sets the z orientation of the 3d scene
+
+:param newOrientationZ: orientation in degrees
+
+.. versionadded:: 3.38
+%End
+
+    double orientation() const;
+%Docstring
+Returns the z orientation of the 3d scene
+
+.. versionadded:: 3.38
+%End
+
+    QgsGeometry orientedExtent() const;
+%Docstring
+Returns the 3D scene's 2D geometry in project's CRS.
```

Comment:
Is this correct? Is it actually project crs, even if the 3d map settings object has a different crs() set? Or should this be "in the map's crs()"? (also applies to the other extent methods).

Reply:
This PR does not change any CRS related thing. I have expanded the existing behavior. Indeed, this is not the case anymore since https://github.com/qgis/qGIS/commit/b0d1a4f8b1cc244ccf2bc3b07798f9ee3540072f

This PR fixes the existing documentation: https://github.com/qgis/QGIS/pull/57613